tutorials/introductory/customizing.py

```diff
@@ -12,7 +12,7 @@
 :ref:`matplotlib rc <customizing-with-matplotlibrc-files>` file (which is read
 at startup to configure matplotlib).
 
-There are a number of pre-defined styles provided by matplotlib. For
+There are a number of pre-defined styles `provided by Matplotlib`_. For
```

Comment:
Can you somehow make clear that these styles are already built in and the link is just for reference. Otherwise, I fear that people will think that they have to download the linked styles and put them somewhere locally.